Kernel/Devices/USB/USBDevice.cpp

```diff
@@ -0,0 +1,76 @@
+/*
+ * Copyright (c) 2021, Jesse Buhagiar <jooster669@gmail.com>
+ *
+ * SPDX-License-Identifier: BSD-2-Clause
+ */
+
+#include <AK/OwnPtr.h>
+#include <AK/Types.h>
+#include <AK/Vector.h>
+#include <Kernel/Devices/USB/UHCIController.h>
+#include <Kernel/Devices/USB/USBDescriptors.h>
+#include <Kernel/Devices/USB/USBDevice.h>
+#include <Kernel/Devices/USB/USBRequest.h>
+
+static u32 s_next_usb_address = 1; // Next address we hand out to a device once it's plugged into the machine
+
+namespace Kernel::USB {
+
+void USBDevice::create(PortNumber port, DeviceSpeed speed)
```

Comment:
The word `create` is used throughout the codebase for functions that create an object and return it. This does something else, so can we find another name?

Reply:
I've removed this function and put all of it inside `try_create` instead to the hotplug daemon can report any errors.